addons/web_editor/static/src/js/frontend/loader.js

```diff
@@ -3,7 +3,7 @@ odoo.define('web_editor.loader', function (require) {
 
 var ajax = require('web.ajax');
 
-let wysiwygPromise;
+let wysiwygPromise = {};
```

Comment:
My initial thought was that this code doesn't do what it's supposed to do: Load a wysiwyg class from its bundle.

Let's say we have 2 wysiwyg defined : web_editor and website.
Let's say we separate them in different bundles: web_editor.assets_wysiwyg and website.assets_wysiwyg.

Now if I load the wysiwyg for the HTMLField I will load all the dependencies for web_editor.wysiwyg, and the promise will be fulfilled.

If I switch to website and try to load its wysiwyg and bundles: the promise is fulfilled but website.assets_wysiwyg is never loaded so it will load something that is not defined, and crash.

The reason it works for mass_mailing is that the wysiwyg for mass_mailing is defined in the same bundle as web_editor, but this isn't the case for website.

That's the reason of the change but maybe my approach is wrong and the bundle should be rethought.
We tried to bypass this function using web's `loadBundle` function with @younn-o but this doesn't add the templates to the legacy env.

Reply:
Ok, I guess we'll have to review the assets and stuff once common will be dropped and once we'll start optimizing before 16.0 release. For the merge, at least rename the variable please :+1: